manifests/mod/cache_disk.pp

```diff
@@ -0,0 +1,91 @@
+# @summary
+#   Installs and configures `mod_cache_disk`.
+#
+# @description
+#   This will install an configure the proper module depending on the used apache version, so
+#   - mod_cache_disk for apache version >= 2.4
+#   - mod_disk_cache for older apache versions
+# 
+# @param cache_root
+#   Defines the name of the directory on the disk to contain cache files.
+#   Default depends on the Apache version and operating system:
+#   - Debian: /var/cache/apache2/mod_cache_disk
+#   - FreeBSD: /var/cache/mod_cache_disk
+#   - Red Hat, Apache 2.4: /var/cache/httpd/proxy
```

Comment:
You can assume Apache 2.4 only and I don't see Apache 2.2 references in this class